roles/migrationcontroller/tasks/mcg.yml

```diff
@@ -0,0 +1,101 @@
+- name: Checking if NooBaa CRDs exist
+  k8s_facts:
+    api_version: "apiextensions.k8s.io/v1beta1"
+    kind: CustomResourceDefinition
+    name: "{{ item }}"
+  loop: "{{ noobaa_expected_crds }}"
+  register: noobaa_found_crds
+
+- set_fact: 
+    noobaa_crds_found: "{{ noobaa_found_crds.results|length == noobaa_expected_crds|length }}"
```

Comment:
I just realized that this condition will fail when NooBaa adds more CRDs. It should be >= and not ==